src/test/java/seedu/address/logic/commands/AddTaskCommandTest.java

```diff
@@ -67,26 +66,23 @@ public void equals() {
         AddTaskCommand addTaskDrinkCommand = new AddTaskCommand(taskDrink);
 
         // same object -> returns true
-        assertTrue(addTaskEatCommand.equals(addTaskEatCommand));
+        assertEquals(addTaskEatCommand, addTaskEatCommand);
 
         // same values -> returns true
         AddTaskCommand addTaskEatCommandCopy = new AddTaskCommand(taskEat);
-        assertTrue(addTaskEatCommand.equals(addTaskEatCommandCopy));
-
-        // different types -> returns false
-        assertFalse(addTaskEatCommand.equals(1));
+        assertEquals(addTaskEatCommand, addTaskEatCommandCopy);
```

Comment:
I think this part can be assertNotEquals(addTaskEatCommand, 1) since its checking for different types

Reply:
I think after PR, can individually add this part back if we feel is necessary

Reply:
The `assertFalse(addTaskEatCommand.equals(1));` will guarantee a true because of object and primitive comparation, a warning is trigger by IDE because of this, thus I remove this test case.

Reply:
ohh I see!